createLogging.js

```diff
@@ -28,7 +28,7 @@ module.exports = function createLogging({
   Transport = transports.Console,
 } = {}) {
   const container = new Container();
-  const TEST = Env.get('NODE_ENV', '') === 'test';
+  const silenceLogger = Env.get.boolish('FOURC_SILENCE_LOGGING', true);
```

Comment:
```suggestion
  const silenceLogger = Env.get.boolish('FOURC_SILENCE_LOGGING', Env.get('NODE_ENV', '') === 'test');
```

was thinking more this

Reply:
Hmm it seems OK to decouple the log silencing logic from NODE_ENV and default to true to suppress (as is the current behavior). Only downside I think is for people that actually set `NODE_ENV=test` to suppress logging which would be the odd case I think.

Reply:
Actually, never mind, the default was to *not* suppress unless NODE_ENV==='test'. I thought the original suppressed by default. So this looks good I think--it should preserve existing behavior and allow for more explicit suppression via the new `FOURC_SILENCE_LOGGING` env var.